.vscode/launch.json

```diff
@@ -18,6 +18,17 @@
                 // Enable break on exception when debugging tests (see: tests/conftest.py)
                 "PYTEST_RAISE": "1",
             },
-        }
+        },
+        {
+            "name": "Python Debugger: Virtac",
+            "type": "debugpy",
+            "request": "launch",
+            "program": "/venv/bin/virtac",
+            "console": "integratedTerminal",
+            "env": {
+                "EPICS_CA_SERVER_PORT": "7064",
+                "EPICS_CA_REPEATER_PORT": "7065",
```

Comment:
This is fine for now, but I think @MJGaughran and I should have a discussion about whether we can make the changeports script available in our dev containers in future